core/config/fungibleConfig.js

```diff
@@ -0,0 +1,23 @@
+import harden from '@agoric/harden';
+
+import { noCustomization } from './noCustomization';
+import { makeCoreMintKeeper } from './coreMintKeeper';
+import { makeNatAssay } from './assays';
+
+// Non-fungible tokens (our default for mints) do not customize
```

Comment:
Should this be "Fungible tokens", or am I misunderstanding?

Reply:
Ah! Good catch, you are right